cohort-parent/pom.xml

```
@@ -137,6 +137,31 @@
 
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
+				<groupId>ca.uhn.hapi.fhir</groupId>
+				<artifactId>hapi-fhir-client</artifactId>
+				<version>5.2.0</version>
```

since these are all the same version, I would recommend using a `maven` property (_e.g._ `watson.buildtools.version`)

```suggestion
				<version>${hapi.fhir.version}</version>
```